mergify/datadog_checks/mergify/config_models/__init__.py

```diff
@@ -1,3 +1,7 @@
+# (C) Datadog, Inc. 2023-present
```

Comment:
Could you revert all the changes to the files in the config_models folder? It is not needed right now, even if the CI fails atm. We'll fix that, sorry again for the noise